CharlieBackend.Business/Services/SecretaryService.cs

```diff
@@ -151,23 +151,42 @@ public async Task<Result<IList<SecretaryDto>>> GetActiveSecretariesAsync()
             return Result<IList<SecretaryDto>>.GetSuccess(_mapper.Map<List<SecretaryDto>>(secretaries));
         }
 
-        public async Task<Result<SecretaryDto>> DisableSecretaryAsync(long secretaryId)
+        public async Task<Result<bool>> DisableSecretaryAsync(long secretaryId)
         {
             var accountId = await GetAccountId(secretaryId);
 
             if (accountId == null)
             {
-                return Result<SecretaryDto>.GetError(ErrorCode.NotFound, "Unknown secretary id.");
+                return Result<bool>.GetError(ErrorCode.NotFound, "Unknown secretary id.");
             }
 
-            var secretary = await GetSecretaryByAccountIdAsync((long)accountId);
+            var disable = await _accountService.DisableAccountAsync((long)accountId);
 
-            if (!await _accountService.DisableAccountAsync((long)accountId))
+            if (!disable)
             {
-                return Result<SecretaryDto>.GetError(ErrorCode.NotFound,"This secretsry account is already disabled.");
+                return Result<bool>.GetError(ErrorCode.Conflict,"This secretsry account is already disabled.");
             }
 
-            return Result<SecretaryDto>.GetSuccess(secretary.Data);
+            return Result<bool>.GetSuccess(disable);
+        }
+
+        public async Task<Result<bool>> EnableSecretaryAsync(long secretaryId)
+        {
+            var accountId = await GetAccountId(secretaryId);
+
+            if (accountId == null)
+            {
+                return Result<bool>.GetError(ErrorCode.NotFound, "Unknown secretary id.");
+            }
+
+            var enable = await _accountService.EnableAccountAsync((long)accountId);
```

Comment:
Perhaps `changedToEnabled` would be a more descriptive name for this `bool` variable.
( `enable` -> `changedToEnabled` )

Reply:
Solved in every Service. 
https://github.com/ita-social-projects/WhatBackend/pull/420/commits/0828ada45c4cfffe03a04ef81e5b5e8606b96148